gitter/pos_tagger.py

```diff
@@ -0,0 +1,28 @@
+import nltk
+
```

Comment:
all nlp files should go in `gitter/nlp/`, and nlp training data files in `gitter/nlp/training_data/`